substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/MethodEntry.java

```diff
@@ -29,13 +29,18 @@
 public class MethodEntry extends MemberEntry {
     TypeEntry[] paramTypes;
     String[] paramNames;
+    String[] paramTypeNames;
```

Comment:
I'm not sure why you are materialising array paramTypeNames here rather than retrieve the names from the respective element of paramTypes. It seems an unnecessary waste of space to avoid a single field indirection. Or is there a good reason for this chnage?

Reply:
Ah ok, I see this is because you need to use it to implement Range::getParamTypeNames. Ok.

Reply:
Me neither, I think at the point of adding it I was using it in more places. Removed.